### PR TITLE
Update SMBEnumeration.tsx

### PR DESCRIPTION
**Pull request by ZIHRAN1**
In this commit, I made several updates to enhance consistency and clarity. I changed the labels in the NativeSelect components from "Scan Speed" to "Scan speed" and from "SMB Script" to "SMB script" to maintain lowercase formatting consistency. I removed redundant references to the UserGuide component and imported and integrated the InstallationModal component, adjusting its usage accordingly. The button text was updated from "Scan" to "Start {title}" to better align with its functionality. Additionally, I improved comments and documentation for clarity, ensuring better maintainability, cleaned up unused variables and imports, and standardized form labels and descriptions for an improved user experience.

### CODE REVIEW

File: src/components/SMBEnumeration/SMBEnumeration.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { Button, LoadingOverlay, NativeSelect, Stack, TextInput } from "@mantine/core";
+import { Button, NativeSelect, Stack, TextInput } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useCallback, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
-import { UserGuide } from "../UserGuide/UserGuide";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import InstallationModal from "../InstallationModal/InstallationModal"; // Added import
```
Comment (basoreeper3007) [inline src/components/SMBEnumeration/SMBEnumeration.tsx:8]:
```suggestion
import InstallationModal from "../InstallationModal/InstallationModal";

File: src/components/SMBEnumeration/SMBEnumeration.tsx [hunk 2]
```diff
@@ -185,38 +185,53 @@ const SMBEnumeration = () => {
...
+                {/* Scan Speed dropdown */}
                 <NativeSelect
-                    label={"Scan Speed"}
+                    label={"Scan speed"}
                     value={selectedSpeedOption}
                     onChange={(e) => setSelectedSpeedOption(e.target.value)}
                     title={"Scan speed"}
                     data={speeds}
-                    placeholder={"Select a scan speed. Default set to T3"}
+                                        placeholder={"Select a scan speed. Default set to T3"}
```
Comment (basoreeper3007) [inline src/components/SMBEnumeration/SMBEnumeration.tsx:208]:
```suggestion
placeholder={"Select a scan speed. Default set to T3"}

File: src/components/SMBEnumeration/SMBEnumeration.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 import { useForm } from "@mantine/form";
 import { useCallback, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
-import { UserGuide } from "../UserGuide/UserGuide";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import InstallationModal from "../InstallationModal/InstallationModal"; // Added import
 
 const title = "SMB Enumeration"; // Title of the component.
```
Comment (basoreeper3007) [inline src/components/SMBEnumeration/SMBEnumeration.tsx:10]:
```suggestion
const title = "SMB Enumeration";

**Review by basoreeper3007 — CHANGES_REQUESTED** [review]
Nice code, only issues are with some indentation, style, and comments. Some inline comments were unnecessary so I have suggested some changes. 

Also please make sure in the future you commit with smaller commits that reflect smaller portions of code changes at a time. Makes it harder to understand the changes you've made when you commit the whole finished code in one go.

After making my suggested changes please run "npx prettier" command on your code for styling the code file. You need to run thisAfter making my suggested changes please run "npx prettier" command on your code for styling the code file. You need to run this so the code can pass the automatic checks.  so the code can pass the automatic checks. If you're not sure how to do that just reach out.

**Comment by basoreeper3007** [comment]
@ZIHRAN1 instead of making the changes i suggested in code and committing them, you should be able to click "commit suggestion" on my comments.

**Comment by basoreeper3007** [comment]
@ZIHRAN1 Still waiting on changes for this before i can approve it.

**Comment by ZIHRAN1** [comment]
Hi, i’ve done the changes.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by basoreeper3007 — COMMENTED** [review]
It wasn't reflecting your changes properly for me. I refreshed and I can see that all changes have been made.

**Comment by basoreeper3007** [comment]
To pass the style check, you need to run prettier on your code.

Running Prettier:
- Navigate to your tsx file in the terminal.
- run the command "npx prettier -w filename.tsx"

**Review by basoreeper3007 — APPROVED** [review]
Good work! I approve.

**Pull request merged**